sell-loot.lic

```diff
@@ -76,11 +76,18 @@ class SellLoot
     fput("open my #{container}")
     return if inside?("#{adj} pouch", container)
     walk_to @hometown['gemshop']['id']
-    @hometown['gemshop']['name'] = "Veilex" if DRRoom.npcs.include?("Veilex") # Shard gemshop changes person manning the shop
-    fput("ask #{@hometown['gemshop']['name']} for #{adj} pouch")
+    clerk = which_clerk(@hometown['gemshop']['name'])
+    fput("ask #{clerk} for #{adj} pouch")
     fput("put my pouch in my #{container}")
   end
 
+  def which_clerk(clerks)
+    return clerks if clerks.is_a? String
+
+    clerks.select! { |clerk| DRRoom.npcs.include?(clerk) }
```

Comment:
These two lines can be replaced with `clerks.find { |clerk| DRRoom.npcs.include?(clerk) }`